pkg/helm/handlers/handler_test.go

```diff
@@ -700,6 +706,29 @@ func TestHelmHandlers_Index(t *testing.T) {
 					},
 				},
 			},
+			httpStatusCode: http.StatusOK,
+		},
+		{
+			name: "valid repo index file should return correct response and also validate onlyCompatible flag",
```

Comment:
"valid repo index file should contain only entries compatible with the given cluster"

Reply:
Done